backend/lib/src/services/msp.rs

```diff
@@ -313,31 +307,17 @@ impl MspService {
         Ok(expected)
     }
 
-    /// Distribute a file to BSPs
-    pub async fn distribute_file(
-        &self,
-        _bucket_id: &str,
-        file_key: &str,
-    ) -> Result<DistributeResponse, Error> {
-        // Mock implementation
-        Ok(DistributeResponse {
-            status: "distribution_initiated".to_string(),
-            file_key: file_key.to_string(),
-            message: "File distribution to volunteering BSPs has been initiated".to_string(),
-        })
-    }
-
     /// Get all payment streams for a user
     pub async fn get_payment_streams(
         &self,
-        user_address: &str,
+        user_address: &Address,
     ) -> Result<PaymentStreamsResponse, Error> {
         debug!(target: "msp_service::get_payment_streams", user = %user_address, "Getting payment streams");
 
         // Get all payment streams for the user from the database
         let payment_stream_data = self
             .postgres
-            .get_payment_streams_for_user(user_address)
+            .get_payment_streams_for_user(&user_address.to_string())
```

Comment:
Changing `user_address` downstream to also be `Address` instead `String` was a PITA?